.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vendor/rubocop"]
```

Comment:
Yes, and I meant to make a change to the README as well. Makes much more sense to just add rubocop as a submodule than to require the user to explicitly `git clone` the latest master and hope nothing breaks.